src/components/ConfirmationFolder/ConfirmationPage.jsx

```diff
@@ -1,25 +1,28 @@
-import React from 'react';
-import './ConfirmationPage.css';
+import "./ConfirmationPage.css";
 
-const ConfirmationScreen = () => {
+export const ConfirmationScreen = () => {
```

Comment:
don't change those because then we have to change the import path in other files as well and this leads to other conflicts - or if you want the named export then change it in other files the file in imported to